glossary.md

```diff
@@ -201,6 +201,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 ## L
 | English                            | Tiếng Việt                     | Thảo luận tại                                |
 |------------------------------------|--------------------------------|----------------------------------------------|
+| label smoothing                    | làm phẳng nhãn                 |                                              |
```

Comment:
`làm mượt` mn nghĩ sao?
```suggestion
| label smoothing                    | làm mượt nhãn                 |                                              |
```

Reply:
+1, từ `smooth` trong ngữ cảnh toán học cũng được dịch như vậy rồi thì phải

Reply:
label dùng sunsilk óng mượt rạng ngời sao...